node_package/src/react_on_rails.js

```diff
@@ -0,0 +1,277 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+
+const context = ((typeof window !== 'undefined') && window) ||
+  ((typeof global !== 'undefined') && global) || this;
+
+let ReactOnRails;
+
+const turbolinksInstalled = (typeof Turbolinks !== 'undefined');
+
+function provideClientReact() {
+  if (typeof context.ReactDOM === 'undefined') {
+    if (React.version >= '0.14') {
```

Comment:
@alexfedoseev assume `React.version == '1.4.3'`, it seems (`React.version >= 0.14`) will return `false`, but (`React.version >= '0.14'`) will return `true`.